.editorconfig

```diff
@@ -135,6 +135,21 @@ csharp_style_inlined_variable_declaration = true:suggestion
 csharp_style_throw_expression = true:suggestion
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/cmdlet-example/command-line-simple-example.md.https://www.microsoft.com/net/core#linuxubuntu should work
```powershell
Failed to complete request to "https://www.microsoft.com/net/core#linuxubuntu". Response status code does not indicate success: 400 (Bad Request).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/building/windows-core.md.https://www.microsoft.com/net/core#windowscmd should work
```powershell
Failed to complete request to "https://www.microsoft.com/net/core#windowscmd". Response status code does not indicate success: 400 (Bad Request).
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 119
119:                                     throw "Failed to complete request to `"$url`". $($_.Exception.Message)"

```